Services/Ordering/Ordering.Domain/ValueObjects/Price.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace Ordering.Domain.ValueObjects
+{
+    public class Price
+    {
+        private decimal _value;
```

Comment:
FYI, this only makes sense if you're going to extend Price class (for example, with currency). Otherwise it better be simplified to decimal